openapi/src/test/groovy/io/micronaut/openapi/visitor/OpenApiIncludeVisitorSpec.groovy

```diff
@@ -29,24 +31,10 @@ import io.swagger.v3.oas.annotations.responses.ApiResponse;
                 license = @License(name = "Apache 2.0", url = "https://foo.bar"),
                 contact = @Contact(url = "https://gigantic-server.com", name = "Fred", email = "Fred@gigagantic-server.com")
         ),
-        tags = {
-                @Tag(name = "Tag 1", description = "desc 1", externalDocs = @ExternalDocumentation(description = "docs desc")),
-                @Tag(name = "Tag 2", description = "desc 2", externalDocs = @ExternalDocumentation(description = "docs desc 2")),
-                @Tag(name = "Tag 3")
-        },
         externalDocs = @ExternalDocumentation(description = "definition docs desc"),
         security = {
                 @SecurityRequirement(name = "req 1", scopes = {"a", "b"}),
                 @SecurityRequirement(name = "req 2", scopes = {"b", "c"})
-        },
-        servers = {
-                @Server(
-                        description = "server 1",
-                        url = "https://foo",
-                        variables = {
-                                @ServerVariable(name = "var1", description = "var 1", defaultValue = "1", allowableValues = {"1", "2"}),
-                                @ServerVariable(name = "var2", description = "var 2", defaultValue = "1", allowableValues = {"1", "2"})
-                        })
```

Comment:
Removed because this test doesn't check anything related to `servers` (that is done in other tests)